README.md

```diff
@@ -10,37 +10,46 @@ For more information, read the [Helm Broker documentation](https://kyma-project.
 
 To run the project, download these tools:
 
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-* [Helm CLI](https://github.com/kubernetes/helm#install)
-* [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) for local installation
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.16
+* [Helm CLI](https://github.com/kubernetes/helm#install) 2.14
+* [Docker](https://docs.docker.com/install/) 19.03 (for local installation)
+* [Kind](https://github.com/kubernetes-sigs/kind#installation-and-usage) 0.5 (for local installation) 
+
+>**NOTE:** For non-local installation, use Kubernetes v1.15.
 
 ## Installation 
```

Comment:
We should somewhere describe that you can just execute the
`./hack/run-dev-kind.sh` which will execute all those steps for you. **Now it is a piece of secret knowledge.**

IMO here we can have just info about this script with prerequisite and in the directory where this `./hack/run-dev-kind.sh` have details what is done under the hood and if someone wants to do it manually then can do it.